views/homepage.twig

```diff
@@ -27,7 +27,7 @@
 
                     {#<br><h3>{{ month|date('F Y') }}</h3>#}
 
-                    {% for talk in talks %}
+                    {% for key, talk in talks %}
```

Comment:
It's used for when there are more than one talk on the same date, to provide a link to the correct talk. I had broken this accidentally but it didn't come up because we haven't had more than one talk for a while.

the keys should really be something a little more seo friendly than /talks/2016/october/1